internal/cmd/config/set/set.go

```diff
@@ -42,22 +42,17 @@ func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "set",
 		Short: "Sets CLI configuration options",
-		Long: fmt.Sprintf("%s\n%s\n%s\n%s",
-			"Sets CLI configuration options.",
-			"All of the configuration options can be set using an environment variable, which takes precedence over what is configured.",
-			`The environment variable is the name of the flag, with underscores ("_") instead of dashes ("-") and the "STACKIT" prefix.`,
-			"Example: to set the project ID you can set the environment variable STACKIT_PROJECT_ID.",
-		),
-		Args: args.NoArgs,
+		Long:  "Sets CLI configuration options.",
```

Comment:
I think the information that was here could still be useful.
Even more if we added the note that this command writes **to the config file**, and that those values can be overritten by env variables

Reply:
Same thing as https://github.com/stackitcloud/stackit-cli/pull/75#discussion_r1483270839 regarding the config file
I can add that env vars take precedence, but I'm not sure this information should be repeated across multiple commands. And where that matters is in `stackit config list`, where the precedence is solved

Reply:
I see the point, but I think it could also be relevant to note here that any value set using this command is not guaranteed to take effect (if there is an env var overriding it)

Reply:
We could also not mention the config file specifically, but I think we should at least mention the possibility that the configuration values set here are being overitten by an env var

Reply:
Re-added